src/core/execute.c

```diff
@@ -2446,8 +2447,12 @@ static int setup_keyring(
 
         /* And now, make the keyring owned by the service's user */
         if (uid_is_valid(uid) || gid_is_valid(gid))
-                if (keyctl(KEYCTL_CHOWN, keyring, uid, gid, 0) < 0)
-                        return log_unit_error_errno(u, errno, "Failed to change ownership of session keyring: %m");
+                if (keyctl(KEYCTL_CHOWN, keyring, uid, gid, 0) < 0) {
+                        log_error_errno(errno, "Failed to change ownership of session keyring: %m");
```

Comment:
this should be downgraded to warning, really. LOG_ERR is what we should use for "fatal" errors, and "warning" for errors we proceed anyway with.

Reply:
yes